.changeset/chilly-dryers-begin.md

```diff
@@ -0,0 +1,47 @@
+---
+"@opennextjs/cloudflare": patch
+---
+
+introduce new `initOpenNextCloudflareForDev` utility and make `getCloudflareContext` synchronous
+
+introduce a new `initOpenNextCloudflareForDev` function that when called in the [Next.js config file](https://nextjs.org/docs/app/api-reference/config/next-config-js) integrates the Next.js dev server with the open-next Cloudflare adapter. Most noticeably this enables `getCloudflareContext` to work in
```

Comment:
suggestion: maybe start simple with the TL;DR and add details (or not) in following paragraph.

something like

```
`initOpenNextCloudflareForDev()` must be called from the [Next.js config file](https://nextjs.org/docs/app/api-reference/config/next-config-js) so that your bindings can be accessed  using `getCloudflareContext()` when running the Next.js development server (`next dev`).

`getCloudflareContext` is now sync.

Reply:
the changeset title is already a good TL;DR, isn't it?

I think I don't fully understand what you're suggesting, if you want feel free to suggest a code change or to push on the branch

Reply:
I think the change set should first highlight what is most important for the users:

1. that the init function must be called from the config
2. that getCloudflareContext is sync

IMO mentioning the edge runtime of our recommendation could be pushed down or moved to the doc site.

Reply:
I've updated the changeset I hope the new version works for you